raving-remoras/bangazon-workflow
# Km past training

# Description
Updates seeder file structure to create both upcoming and past training sessions and assigns employees to each session. 

## Steps to Test Solution

1. In django_data.sh file add `python manage.py $3` to the end of the script and save
1. Pull down the file and type the command `django_data.sh agileHR seeder training_seeder`
1. This should allow you to build a new database with upcoming and past training and have employees assigned to the training programs

## Documentation
